app/views/insecure_password/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :title, t("insecure_password.interstitial.heading") %>
+
+<%= render "govuk_publishing_components/components/heading", {
+  text: yield(:title),
+  heading_level: 1,
+  font_size: "xl",
+  margin_top: 0,
```

Comment:
`margin_top` is not an option on this component, this line can be removed.

Reply:
We've got a bunch of these in different pages, I'll push a commit removing them all